_topic_maps/_topic_map.yml

```diff
@@ -2316,6 +2316,10 @@ Topics:
   File: hcp-managing
 - Name: Using feature gates in a hosted cluster
   File: hcp-using-feature-gates
+- Name: Updating hosted control planes
+  File: hcp-updating
+- Name: Hosted control planes Observability
```

Comment:
Should "observability" be capitalized here? It's not capitalized in the module heading.